### DISCUSSION TITLE
OIDCUnAuthzAction typo?

### DISCUSSION THREAD

**Discussion by lhoekenga** [OpenIDC/mod_auth_openidc#1073] [General]
The code / documentation call the the directive `OIDCUnAutzAction`
The ChangeLog and the  [release notes for 2.4.14.2 ](https://github.com/OpenIDC/mod_auth_openidc/releases/tag/v2.4.14.2) call it `OIDCUnAuthzAction`

I feel like it should be consistent

**Comment by zandbelt** [comment]
that is fixed now, thanks!
